infrastructure/modules/aws/eks/versions.tf

```diff
@@ -2,19 +2,19 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.18.0"
+      version = ">= 4.47.0"
     }
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = ">= 2.7.1"
+      version = "~> 2.13.0"
```

Comment:
Prefer `>=`: https://developer.hashicorp.com/terraform/language/expressions/version-constraints#terraform-core-and-provider-versions

Reply:
I agree